gitlab/v4/objects/events.py

```diff
@@ -110,6 +124,18 @@ class ProjectMergeRequestResourceMilestoneEventManager(RetrieveMixin, RESTManage
     _from_parent_attrs = {"project_id": "project_id", "mr_iid": "iid"}
 
 
+class ProjectMergeRequestResourceStateEvent(RESTObject):
+    pass
+
+
+class ProjectMergeRequestResourceStateEventManager(RetrieveMixin, RESTManager):
+    _path = (
+        "/projects/%(project_id)s/merge_requests/%(issue_iid)s/resource_state_events"
```

Comment:
Shouldn't this be 

```
"/projects/%(project_id)s/merge_requests/%(mr_iid)s/resource_state_events"
```

like here? https://github.com/python-gitlab/python-gitlab/pull/1364/commits/40eb59e66ad381bd329a66d9e69db61b10aa5440#diff-9715eb24ecd3f0c1c56e83d4867974145fcffdf50cabb59042173492bb440274R134

Reply:
Copy/paste :facepalm: Done, thanks :grin: